### ISSUE TITLE
Avoid use of "proprietary" when speaking of rights infringements

### ISSUE THREAD

**Issue by ossguy** [github/site-policy#72]
#### 1. What's the name of the policy?

https://github.com/github/site-policy/blob/master/Policies/github-terms-of-service.md

#### 2. Is this issue related to a specific section within one of our policies (e.g. the Terms of Service)? If so, please include a link to the section or subsection.

https://github.com/github/site-policy/blob/master/Policies/github-terms-of-service.md#2-content-restrictions

#### 5. Why do you think this section or language needs improvement?

It is not immediately clear what the term "proprietary right" means, since "proprietary" is often used as a way for companies to describe rights that they think they have (but might not be supported by actual written or case law).  As a result, the last bullet point could be construed to mean that the user should never infringe on any right that a company or other user thinks they have (such as an action clearly allowed by fair use, but that the company or other user doesn't want you to engage in).

So instead of "proprietary right", it should say "valid legal right" or similar instead.  This will hopefully clear up the confusion.

**Comment by nsqe** [comment]
Hey, @ossguy — "proprietary rights" are actually very specific things, and refer to rights of ownership, specifically what we think of as intellectual property rights. "Valid legal rights" doesn't mean the same thing, and would be overbroad. While we have tried very hard, generally, to write our Terms of Service in layman's terms, it's still a legal contract and sometimes we have to use the right legal language to describe what we mean.

So here, when we say _"infringes on any proprietary right of any party,"_ we clarify it by spelling out what we mean by proprietary rights: _"including patent, trademark, trade secret, copyright, right of publicity, or other rights."_

**Comment by wking** [comment]
On Fri, Jul 28, 2017 at 02:24:04PM -0700, Hannah Poteat wrote:
> "Valid legal rights" doesn't mean the same thing, and would be overbroad.

Moral rights seem to be a non-proprietary legal right.  E.g. from [1]:

  It comprises not only proprietary rights but also moral rights.

If §C.2 [2] is explicitly excluding moral rights, is content that
violates moral rights (but not proprietary rights) considered
exceptable?  It seems like the wider “valid legal rights” would be
more appropriate.

[1]: http://www.lexology.com/library/document.ashx?g=365c8b4a-efe5-45c7-872f-88ca68042d7a#page=9
[2]: https://github.com/github/site-policy/blob/388f344f4e30a2840e6d27d2854e05d7e8e5e287/Policies/github-terms-of-service.md#2-content-restrictions
